composer.json

```diff
@@ -5,9 +5,8 @@
     "license": "MIT",
     "require": {
         "php": ">=5.4.0",
-        "react/socket": "0.4.*",
-        "react/dns": "0.4.*",
-        "react/socket-client": "0.4.*"
+        "react/socket": "0.8.*",
```

Comment:
As per https://github.com/reactphp/socket/issues/75, you may as well target future versions with `^1.0 || ^0.8` here :+1: